pkg/workflow/providers/multicluster/deploy_test.go

```diff
@@ -83,31 +83,47 @@ func TestOverrideConfiguration(t *testing.T) {
 
 func TestApplyComponents(t *testing.T) {
 	r := require.New(t)
-	const n, m = 50, 5
+	const n, m = 30, 5
```

Comment:
Why reduce it to 30? I think this test is fast?

Reply:
Misunderstood this number. I thought it should be divided by 3 exactly.

Reply:
n should be divided by 3 exactly to pass the designed test.